src/hooks/usePointerState.ts

```diff
@@ -6,25 +6,29 @@ export default function usePointerState() {
   const [pressActive, setPressActive] = useState(false) // Touch
 
   const onMouseEnter = () => setHover(true)
-  const onMouseLeave = () => setHover(false)
+  const onMouseLeave = () => {
+    if (pressActive) setPressActive(false)
+    setHover(false)
+  }
 
   const handlePress = (e: PointerEvent<HTMLDivElement>) => {
```

Comment:
Can you address this unused parameter so I can access the  netlify preview for a clickthrough?